invenio_i18n/__init__.py

```diff
@@ -133,13 +129,47 @@
 or the ``{% trans %}`` tag:
 
 >>> with app.app_context():
-...     r = render_template_string("{% trans %}Long translation{% endtrans %}")
-...     r == "Long translation"
+...     r = render_template_string('{% trans %}Long translation{% endtrans %}')
+...     r == 'Long translation'
 True
 
 For further details and examples see:
 
 * http://jinja.pocoo.org/docs/dev/templates/#i18n
+
+Templates
+---------
+
+Filters
```

Comment:
Fair point. I just wanted to highlight the fact that they are automatically registered and people can use them. I will add links to API docs.